### ISSUE TITLE
Introducing tests

### ISSUE THREAD

**Issue by adri326** [pony-bio/pony-bio#14]
As soon as the foundations of `pony-bio` become somewhat stable, automated tests should be introduced.

**Comment by rhagenson** [comment]
Absolutely! I think once the `Sequence` class is added (#13) that would be a good time to start tackling tests as that will be the first instance of supporting common user-defined input. I would prefer using [`ponycheck`](https://github.com/mfelsche/ponycheck) over `ponytest` where possible so (much like I did in [sembio/go](https://github.com/sembio/go)) the tests act as a sort of documentation of expectations until real documentation is created. I then see the value of unit tests via `ponytest` as being for checking known edge cases which may or may not be invoked during a given `ponycheck` run.
